devdoc.md

```diff
@@ -0,0 +1,165 @@
++ Developers who are members of the [Arcadia-Science](https://github.com/Arcadia-Science/) organization: clone the repository and create a branch.
+Clone the repository locally using `git clone`.
+
+See [contributing section](#contributing-changes-back-to-the-sourmashconsumr-repository) below for instructions on how to contribute code changes back to the sourmashconsumr repository.
+
+### Installing R and RStudio
+
+Make sure you have R and RStudio installed (see [here](https://rstudio-education.github.io/hopr/starting.html) for operating-system specific instructions).
+
+### Opening the sourmashconsumr R project
+
+You'll then need to open the R project for the sourmashconsumer project. 
+You can do this by double clicking the `sourmashconsumr.Rproj` file in your file finder or by using `File` -> `Open Project...` and then finding and opening the `sourmashconsumr.Rproj` file.
+
+### Installing development dependencies
+
+To start developing the package, you'll need to install `devtools`, `testthat`, and `usethis`. 
+
+You can use the following R command to install these packages:
+
+```
+install.packages(c("devtools", "testthat", "usethis"))
+```
+
+To be able to run all functions, and to run all of the tests, you will also need to install the package dependencies. 
+See the instructions below for options for how to install these dependencies.
+
+### Installing the package dependencies
+
+The sourmashconsumr package has a lot of dependencies as it tries to make the outputs of the sourmash python package interoperable with biological computing packages already in the R ecosystem. You can install these dependencies however you like; they're all documented in the `DESCRIPTION` file in the root of this repository. Below we describe different strategies for installing the dependencies.
+
+#### Installing package dependencies using R functions
+
+Installing the sourmashconsumr package from github will trigger missing dependencies to be installed.
+
+```
+install.packages("devtools")
+devtools::install_github("Arcadia-Science/sourmashconsumr")
+```
+
+Some of the package dependencies are available on Bioconductor (namely phyloseq), so if installation of dependencies fails through `install_github`, you may have more success using `BiocManager`:
+```
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("Arcadia-Science/sourmashconsumr")
+```
+
+#### Managing the development environment with conda
+
+If the above methods fail, conda can be used to install the dependencies.
+You can find operating system-specific instructions for installing miniconda [here](https://docs.conda.io/en/latest/miniconda.html).
+Once you have conda and [mamba](https://mamba.readthedocs.io/en/latest/) installed, you can create an environment with all of the sourmashconsumr dependencies and developer packages installed using the following bash command:
+
+```
+mamba env create -n sourmashconsumr --file devenv.yml
```

Comment:
If I do this do i still have to manually install development dependencies?

Reply:
Nope!